docs/content/preview/explore/ysql-language-features/sql-feature-support.md

```diff
@@ -16,130 +16,138 @@ To understand which standard SQL features we support, refer to the following tab
 
```

Comment:
Replace sentence with "This page highlights the important differences in feature support between YSQL and SQL.